akka-docs/src/test/scala/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -155,7 +155,7 @@ class TwitterStreamQuickstartDocSpec extends AkkaSpec {
       .filterNot(_.hashtags.contains(akkaTag)) // Remove all tweets containing #akka hashtag
       .map(_.hashtags) // Get all sets of hashtags ...
       .reduce(_ ++ _) // ... and reduce them to a single set, removing duplicates across all tweets
-      .mapConcat(identity) // Flatten the stream of tweets to a stream of hashtags
+      .mapConcat(identity) // Expand the set of hashtags to a stream of hashtags
```

Comment:
I kind of liked 'flatten' here, but otherwise :+1:

Reply:
@pm47 could you change back to Flatten here

Reply:
@patriknw done